json-guides/microprofile-config.json

```diff
@@ -343,7 +343,87 @@
             "title": "Configuring as a system property",
             "TOCIndent": 1,
             "description": [
-                "temp"
+                "In WebSphere Liberty, default configuration variables can be retrieved from the Java System Properties and these variables have a default ordinal of 400. The system properties can be set in both the <code>boostrap.properties</code> and in <code>jvm.options</code>. In this example, we are going to use the <code>bootstrap.properties</code> in the Liberty installation's servers directory under <code>usr/servers/<server_name></code> to configure the download_url system property. The bootstrap.properties file does not exist by default until you create it."
```

Comment:
Liberty adds properties from the server’s bootstrap.properties and jvm.options file to the Java System Properties. For jvm.options file, it could be in multiple places. This is from KC (https://www.ibm.com/support/knowledgecenter/en/was_beta_liberty/com.ibm.websphere.wlp.nd.multiplatform.doc/ae/twlp_admin_customvars.html):
You can create multiple jvm.options files in the following locations:
${wlp.install.dir}/usr/shared/jvm.options
${server.config.dir}/configDropins/defaults/jvm.options
${server.config.dir}/jvm.options
${server.config.dir}/configDropins/overrides/jvm.options
These files are merged in order from top to bottom before server start and are used as the JVM options for the server. If none of these files are present, then ${wlp.install.dir}/etc/jvm.options is read.